pedalboard/ExternalPlugin.h

```diff
@@ -758,6 +763,103 @@ class ExternalPlugin : public AbstractExternalPlugin {
     return mainInputBus->getNumberOfChannels();
   }
 
+  /**
+   * Send a MIDI note into this plugin in an attempt to wait for the plugin to
+   * "warm up". Many plugins do asynchronous background tasks on launch (such as
+   * loading assets from disk, etc). These background tasks may depend on the
+   * event loop, which Pedalboard does not pump by default.
+   *
+   * Returns true iff the plugin rendered audio; false if no audio was received.
+   */
+  bool attemptToWarmUp(int timeoutMilliseconds = 5000) {
```

Comment:
For the future, time values like this should be a [std::chrono::duration](https://en.cppreference.com/w/cpp/chrono/duration) object.